docs/internal/DistributedArchitectureGuide.md

```diff
@@ -12,6 +12,218 @@ A guide to the general Elasticsearch components can be found [here](https://gith
 
 # Networking
 
+Every elasticsearch node maintains various networking clients and servers,
+protocols, and synchronous/asynchronous handling.
+
+## HTTP Transport
+The HTTP Transport Server (simply HTTP Transport) is a single entry point for
+all external clients (excluding cross-cluster communication). Management,
+ingestion, search, and all other external operations pass through the HTTP
+server.
+
+Elasticsearch works over HTTP 1.1 and supports features such as TLS, chunked
+transfer encoding, content compression, and pipelining. While attempting to
+be HTTP spec compliant, Elasticsearch is not a webserver. ES Supports `GET`
+requests with a payload (though some old proxies may drop content) and
+`POST` for clients unable to send `GET-with-body`. Requests cannot be cached
+by middle boxes.
+
+There is no connection limit, but a limit on payload size exists. The default
+maximum payload is 100MB after compression. It's a very large number and almost
+never a good target that the client should approach.
+
+Security features, including basic security (authc/authz/TLS) in the free tier,
```

Comment:
```suggestion
Security features, including basic security (authentication/authorization/Transport Layer Security) in the free tier,
```
For someone that is not used to working with networking and security (me) it's not obvious what authc/authz/TLS means without googling. If target audience is newcomers and open source community I think it's worth spelling those out.